src/i18n/en-US/intake.ts

```diff
@@ -139,6 +139,12 @@ const intake = {
     archivedAt: 'Archived At',
     adminLead: 'Admin Lead'
   },
+  hasContract: {
+    HAVE_CONTRACT: 'Yes',
+    IN_PROGRESS: 'In progress',
+    NOT_STARTED: 'Not started',
+    NOT_NEEDED: 'Not needed'
+  },
```

Comment:
Could we just use [these values](https://github.com/CMSgov/easi-app/blob/167d99aea4b01ebd14b2501136109bdf3668c617/src/constants/enums/contractStatus.ts#L1-L10) instead of making new translations?

I don't want to nit pick it much, but thought it was worth calling out that these enums already have translations (unless there was a need/expectation to translate them differently for the sake of the CSV)

Reply:
I updated the translations. I didn't realize those were there!

Reply:
Thanks for the quick fix!